app/components/Settings/Preview/Preview.react.js

```diff
@@ -16,8 +16,8 @@ export default class Preview extends Component {
         const tableHeaders = columnnames.map(
             column => <th>{column}</th>
         );
-
-        const renderCell = cell => <td>{JSON.stringify(cell)}</td>;
+        const isObject = (arg) => typeof arg === 'object';
+        const renderCell = cell => <td>{isObject(cell) ? JSON.stringify(cell) : cell}</td>;
```

Comment:
geojson and some dates from redshift can be object asfaik right now

I think doing that conversion is still not clear to me as a good idea for the API - depends what role workspace should play in parsing this.